### ISSUE TITLE
3.21.2: Overlays transparency loss

### ISSUE THREAD

**Issue by Moonbase59** [valentine195/obsidian-leaflet-plugin#182]
With 3.21.2, my (previously working) maps suddenly show _much less transparency_ on the overlays:

![Auswahl_076](https://user-images.githubusercontent.com/3706922/127781966-b97912db-694c-464b-b901-d1d1b9a64fcf.png)

The maps have:

```
overlayColor: 'rgba(218, 165, 32, .6)'
```

**Comment by valentine195** [comment]
Can you check if there are actually multiple overlays there?

**Comment by Moonbase59** [comment]
Somehow looks like. I clicked the top one in the lower map, and the marked plus the "light blue" (where the cursor is) seem dups:

![1_077](https://user-images.githubusercontent.com/3706922/127792710-4b12b52a-e5af-4e2e-87e8-e1416d9bc6c1.png)

**Comment by Moonbase59** [comment]
All via other notes, wait, I’ll copy the codeblock for you:

````
## Nearby

People, locations and events within `=[[-Current Location]].nearby` around location `=[[-Current Location]].location` (`=[[-Current Location]].address`).

```leaflet
id: Main Map
markerFolder: People
markerFolder: Locations
markerFolder: Events
#markerFolder: -Inbox
height: 500px
lat: 48.23875
long: 10.37595
minZoom: 1
maxZoom: 18
defaultZoom: 12
unit: km
distanceMultiplier: 1.3
overlayTag: nearby
overlayColor: 'rgba(218, 165, 32, .6)'
coordinates: [[-Current Location]]
zoomTag: nearby
zoomDelta: 1
geojson:
  #- [[geo-bayern.geojson]]
  - [[geo-bayern-landkreise.geojson]]
  #- [[geo-lk-günzburg.geojson]]
  - [[Kneipp-Rundweg Krumbach.geojson]]
  - [[Lehrpfad Krumbach.geojson]]
  - [[Walderlebnispfad Krumbach.geojson]]
  - [[Gewässer-Informations-Erlebnis-Pfad Münsterhausen.geojson]]
  - [[Krumbach Oberes Gemeindeholz & Weiher.geojson]]
  - [[Krumbach Oberes Gemeindeholz.geojson]]
  - [[Kleine Runde Kellerweg.geojson]]
#linksTo: [[note]] #markers from all notes linking TO this note
#linksFrom: [[note]] #markers from all notes linking FROM this note
#verbose: true
```
````

**Comment by valentine195** [comment]
Just to make sure, can you look in your `data.json` file and see if there are any overlays associated with that map id?

**Comment by Moonbase59** [comment]
`data.json.zip` sent. Contains lots of multiple entries.

**Comment by valentine195** [comment]
Fixed in 3.21.3

**Comment by Moonbase59** [comment]
So I started out with a "Main Map" in `data.json` that had (manually) the `overlays` array emptied.

Map worked fine, removed the complete "Main Map" entry.

Added a mutable marker, map shows up correctly in `data.json`.

Removed that, complete map entry vanishes from `data.json`.

All good, thanks!
